nslsii/tests/test_logutils.py

```diff
@@ -57,24 +57,51 @@ def test_configure_bluesky_logging_with_unwriteable_dir(tmpdir):
         configure_bluesky_logging(ipython=ip,)
 
 
-def test_default_bluesky_logging():
+def test_configure_bluesky_logging_creates_default_dir():
     """
     Remove environment variable BLUESKY_LOG_FILE and test that
-    the default log file path is used. This test creates a
+    the default log file path is created. This test creates a
     directory rather than using pytest's tmp_path so the test
     must clean up at the end.
     """
     test_appname = "bluesky-test"
-    user_log_dir = Path(appdirs.user_log_dir(appname=test_appname))
+    log_dir = Path(appdirs.user_log_dir(appname=test_appname))
+    # remove log_dir if it exists
+    # make a reasonable effort to delete the log directory
+    # no heroics
+    if log_dir.exists():
+        for f in log_dir.iterdir():
+            f.unlink()
+        log_dir.rmdir()
```

Comment:
Why not to use https://docs.python.org/3/library/shutil.html#shutil.rmtree here? Seems simpler.